### PR TITLE
Stop using jQuery in addon code

### PR DESCRIPTION
**Pull request by cibernox**
@ef4 It was a little bit more complex than expected because `$.css` does some smart things with numeric properties, but it's passing now.

### CODE REVIEW

File: addon/-private/sprite.js [hunk 1]
```diff
@@ -24,6 +23,22 @@ export const COPIED_CSS_PROPERTIES = [
   'letter-spacing'
 ];
 
+const numericCSSProps = {
```
Comment (cibernox) [inline addon/-private/sprite.js]: This is copied from the source code of jQuery.

File: ember-cli-build.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
   var app = new EmberAddon(defaults, {
+    vendorFiler: { 'jquery.js': null, 'app-shims.js': null }
```
Comment (ef4) [inline ember-cli-build.js]: I think this is a typo for `vendorFiles`?

**Comment by ef4** [comment]
With jQuery really disabled, our demo app won't run because our version of ember-cli-addon-docs is using jQuery. It looks like that is already fixed on master there.

**Comment by ef4** [comment]
I merged all your commits. The one remaining one on this branch that I added (which really disabled jQuery) we can do on a separate PR once we have an updated ember-cli-addon-docs.

Great work!

**Pull request closed without merging**